application/views.py

```diff
@@ -22,79 +24,77 @@ def new_application(request):
     try:
         error_count_app = None
         user = Student.objects.get(username=username)
-        objects = HighSchoolApplication.objects.filter(user=user.pk)
-        if HighSchoolApplication.objects.filter(user=user.pk, is_draft=False):
-            error_apply = "You can only submit all applications at once."
-        else:
-            error_apply = None
+        sub_date = timezone.now() - timedelta(days=180)
```

Comment:
This would not hamper our demo even remotely, but I'm just curious about the choice of 180 days.

Reply:
This can be changed upon discussion